logger.js

```diff
@@ -104,10 +104,10 @@ module.exports = (externalLogger, logFolder, logFilePrefix = "installer")=> {
     setDisplaySettings(settings) {
       if (externalLogger) {externalLogger.setDisplaySettings(settings);}
     },
-    external(evt, detail) {
-      appendToLog(evt, detail);
+    external(evt, detail, table) {
+      appendToLog(table || evt, detail);
```

Comment:
@jungeunyoon This will break `appendToLog()` if *table* is applied. *evt* and *detail* should continue to be passed to `appendToLog()` as normal.